index.html

```diff
@@ -386,10 +386,11 @@ <h2>WordWise</h2>
                 <div class="dropdown">
                     <!-- Profile Icon -->
     
-                        <a href="javascript:void(0);" class="profile-link" style="display: flex; align-items: center; text-decoration: none;" onclick="toggleMenu()">
-                            <img src="https://i.postimg.cc/NFFmtc9K/xnz-Mst5-V-400x400.jpg" alt="Profile Icon" style="width: 50px; height: 50px; border-radius: 50%; margin-top: -17px;">
-                        </a>
-                    </li>
+                    <a href="#" class="profile-icon" id="profileDropdown" onclick="toggleDropdown()">

```

Comment:
keep the anchor tag , just change the style , if you change the class and onClick event ,it will change the js code too